.gitignore

```diff
@@ -1,2 +1,5 @@
 .*.swp
 .vscode
+
+# jupyter book related
+bsmschema/build/
```

Comment:
Isn't this covered in `bsmscheme/.gitignore`?

Reply:
ha 
it is not and definitely better to put it there.